javascripts/discourse/initializers/category-icons.js

```diff
@@ -176,6 +178,42 @@ export default {
           }
         },
       });
+
+      if (api.registerCustomCategorySectionLinkLockIcon) {
```

Comment:
@nattsw I added a conditional here to ensure that this theme remains backwards compatible with older versions of Discourse.